core/src/main/java/tc/oc/pgm/stats/StatsMatchModule.java

```diff
@@ -106,7 +106,7 @@ public void onDamage(EntityDamageByEntityEvent event) {
             + absorptionHearts;
 
     if (damager != null) getPlayerStat(damager).onDamage(realFinalDamage, bow);
-    getPlayerStat(damaged).onDamaged(realFinalDamage);
+    if (damaged != null) getPlayerStat(damaged).onDamaged(realFinalDamage, bow);
```

Comment:
Why this if condition? The condition is checked on line 99 already, so this will always be false.
(Guessing because of old commit)

Reply:
This was part of the original commit yes, I've removed it seeing as it's unnecessary.